### DISCUSSION TITLE
Can I run immich app on this?

### DISCUSSION THREAD

**Discussion by sorinciupitu** [immich-app/immich#5466] [Q&A]
I want to buy this machine: HP T630 Thin Client, AMD GX-420GI 2.00GHz, 8GB DDR4, 32GB SSD and I want to know if this is ok for this application.

**Comment by GuillaumeFX** [comment]
It's good, it might be a little tight with machine learning but on my server at 1.60GHz it's functional for me. I think you can take it. However, run the server on a Debian or Ubuntu without a desktop environment.

**Reply by sorinciupitu** [reply]
thank you!

**Comment by GuillaumeFX** [comment]
You're just going to use the 32GB for the system. You will prefer to connect several disks to raid to store your photos.
